python_modules/dagster/dagster/_core/definitions/assets.py

```diff
@@ -746,7 +753,12 @@ def keys_by_output_name(self) -> Mapping[str, AssetKey]:
 
     @property
     def keys_by_input_name(self) -> Mapping[str, AssetKey]:
```

Comment:
I think there are situations where a caller of a function with this name would expect what's returned here to contain entries for every input, including the subsettable inputs.  Is there some way we can address that issue by e.g. renaming this or handling the subsettable inputs in the callers?

Here's the places where it's used:
```
python_modules/dagster/dagster/_core/definitions/asset_checks.py:        ins={name: AssetIn(key) for name, key in asset_def.keys_by_input_name.items()},
python_modules/dagster/dagster/_core/definitions/assets.py:        return self.keys_by_input_name.keys()
python_modules/dagster/dagster/_core/definitions/resolved_asset_deps.py:        for input_name, upstream_key in assets_def.keys_by_input_name.items():
python_modules/libraries/dagster-airbyte/dagster_airbyte/asset_defs.py:        deps=list((assets_defn_meta.keys_by_input_name or {}).values()),
python_modules/libraries/dagster-dbt/dagster_dbt/cloud/asset_defs.py:            deps=list((assets_definition_cacheable_data.keys_by_input_name or {}).values()),
```

It seems like in a couple of the places we don't even use the input names. Other places might be fine if this includes more entries.

Reply:
looked into this and yeah the callers do want to have the full mapping here. updated